application/server/server_smoke_test.go

```diff
@@ -434,7 +450,7 @@ func checkDiagnosticPublishingForCachingSmokeTest(
 			packageJsonCount == expectedOSS
 
 		return result
-	}, time.Second*5, time.Second)
+	}, time.Second*600, time.Second)
```

Comment:
Is the timeout here due to servers not being available for smoke test runs, or test cases taking a long time to run?

Reply:
yeah, I had timeouts running the tests.